src/org/zaproxy/zap/extension/pscanrulesAlpha/resources/Messages.properties

```diff
@@ -1,6 +1,14 @@
 # This file defines the default (English) variants of all of the internationalised messages
 pscanalpha.desc = Passive Scan Rules - alpha
 
+pscanalpha.cookiesamesitescanner.name = Cookie Without SameSite Attribute
+pscanalpha.cookiesamesitescanner.desc=A cookie has been set without the SameSite attribute, which means that the cookie can be sent as a result of a 'cross-site' request. \
+The SameSite attribute is an effective counter measure to cross-site request forgery, cross-site script inclusion and timing attacks.
+pscanalpha.cookiesamesitescanner.badval=A cookie has been set with an invalid SameSite attribute value, which means that the cookie can be sent as a result of a 'cross-site' request. \
+The SameSite attribute is an effective counter measure to cross-site request forgery, cross-site script inclusion and timing attacks.
+pscanalpha.cookiesamesitescanner.soln=Ensure that the SameSite attribute is set to either 'lax' or ideally 'strict' for all cookies.
+pscanalpha.cookiesamesitescanner.refs=https://tools.ietf.org/html/draft-west-first-party-cookies-07
```

Comment:
Maybe just:
https://tools.ietf.org/html/draft-ietf-httpbis-cookie-same-site

which is redirected to that one (latest revision?).